src/Command/UserLoginCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\User;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Uid\Uuid;
+
+class UserLoginCommand extends Command
+{
+    protected static $defaultName = 'app:user-login';
+    protected static $defaultDescription = 'Get login url for user';
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $urlGenerator;
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    public function __construct(UrlGeneratorInterface $urlGenerator, EntityManagerInterface $entityManager)
+    {
+        $this->urlGenerator = $urlGenerator;
+        $this->entityManager = $entityManager;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription(self::$defaultDescription)
+            ->addArgument('email', InputArgument::REQUIRED, 'Argument description')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $email = $input->getArgument('email');
+
+        if ($email) {
+            $io->note(sprintf('You passed an argument: %s', $email));
+        }
+        // Lookup username in DB, fetch the token and then pass it on
+        $user = $this->entityManager->getRepository(User::class)
+            ->findOneBy(['email' => $email]);
+        if (null === $user) {
+            throw new RuntimeException('User not found in database');
+        }
+        // generate new token and set it on the user
+        $token = Uuid::v4()->toBase32();
+        $user->setLoginToken($token);
+        $this->entityManager->persist($user);
+        $this->entityManager->flush();
+
+        $loginPage = $this->urlGenerator->generate('default', [
+            'loginToken' => $token,
+        ], UrlGeneratorInterface::ABSOLUTE_URL);
+
+        $io->success('URL:   '.$loginPage);
```

Comment:
Consider removing `'URL:   '` to have the url on a line by itself. This makes it easier to copy it (e.g. by tripple clicking the line).